packages/@react-aria/interactions/test/useHover.test.js

```diff
@@ -530,10 +530,7 @@ describe('useHover', function () {
     }
 
     beforeAll(() => {
-      jest.useFakeTimers('legacy');
-    });
-    afterAll(() => {
-      jest.useRealTimers();
+      jest.useFakeTimers();
```

Comment:
Why do we have to have `jest.useFakeTimers` in this `beforeAll()`? The [above](https://github.com/adobe/react-spectrum/pull/4401/files#diff-4bda4b9f9f00c7750e961d7ecb0175317e03373dee048725feed5e38f0d5d8c7R40) `beforeAll()` is scoped to cover this abnd has `jest.useFakeTimers`.

Reply:
no idea, didn't notice it, could probably be removed